packages/vendure-order-client/src/vendure-order-client.ts

```diff
@@ -364,9 +364,36 @@ export class VendureOrderClient<A = unknown> {
       this.queries.GET_ORDER_BY_CODE,
       { code }
     );
+    if (
```

Comment:
GetOrderByCode should not do this check. it should be a dumb function: just return the result. This logic can be added to `pollOrderByCode`: if not settled or authorized, poll again